packages/track-transcript/src/GTEx.js

```diff
@@ -140,26 +145,35 @@ TissueIsoformExpressionPlotHeader.defaultProps = {
 }
 
 
+const WithExpressionTooltip = withTooltip(
+  ({ expression, transcriptId }) => (
+    <span><strong>{transcriptId}:</strong> {expression}</span>
+  )
+)
+
+
 export function TissueIsoformExpressionPlot({
+  currentTissue,
   height,
-  width,
+  maxTissueExpressions,
   transcript,
-  currentTissue,
-  tissueStats,
+  width,
 }) {
-  const allTissues = Array.from(tissueStats.keys()).sort()
-  const maxTissueValue = tissueStats.first()
   const selectedTissue = currentTissue || 'median-across-all'
+  const maxExpressionValue = selectedTissue === 'median-across-all'
+    ? maxTissueExpressions.aggregate.median
+    : maxTissueExpressions.individual[selectedTissue]
+
   const padding = 10
 
   const gtexScale = scaleLinear()
-    .domain([0, maxTissueValue + (maxTissueValue * 0.3)])
+    .domain([0, maxExpressionValue])
```

Comment:
Perhaps make the max transcript value take 90% of the plot (rather than 100%)?

Reply:
With the padding added by `.nice()`, the max expressed transcript's usually won't be flush with the right edge of the plot.

https://github.com/d3/d3-scale#continuous_nice